haskell-build/haskell-build-orb.yml

```diff
@@ -107,6 +107,24 @@ examples:
                   - run: echo "I run after tests"
 
 executors:
+  ghc-7.10.3:
+    docker:
+      - image: quay.io/haskell_works/ghc-7.10.3:latest
+  ghc-8.0.2:
+    docker:
+      - image: quay.io/haskell_works/ghc-8.0.2:latest
+  ghc-8.2.2:
+    docker:
+      - image: quay.io/haskell_works/ghc-8.2.2:latest
+  ghc-8.4.4:
+    docker:
+      - image: quay.io/haskell_works/ghc-8.4.4:latest
+  ghc-8.4.3:
```

Comment:
Does it work with `.` in the executor name? I thought I tried and it didn't work, and that's why I updated them to `ghc-863`. But if it works for you then it must have been something else..

Reply:
No, it doesn't work this way:
```
$ circleci orb validate haskell-build-orb.yml
Error: ERROR IN CONFIG FILE:
[#/executors] 6 schema violations found
1. [#/executors/ghc-8.0.2] string [ghc-8.0.2] does not match pattern ^[a-z][a-z\d_-]*$
2. [#/executors/ghc-8.2.2] string [ghc-8.2.2] does not match pattern ^[a-z][a-z\d_-]*$
3. [#/executors/ghc-7.10.3] string [ghc-7.10.3] does not match pattern ^[a-z][a-z\d_-]*$
4. [#/executors/ghc-8.4.3] string [ghc-8.4.3] does not match pattern ^[a-z][a-z\d_-]*$
5. [#/executors/ghc-8.4.4] string [ghc-8.4.4] does not match pattern ^[a-z][a-z\d_-]*$
6. [#/executors/ghc-8.6.3] string [ghc-8.6.3] does not match pattern ^[a-z][a-z\d_-]*$
```

Reply:
That's annoying.  Can we put underscores instead?

Reply:
I've updated to use underscores instead.

Reply:
Cool! Can you also update the "examples" section?
I see that the old `844` and `863` are still there, have you decided to keep them for compatibility reasons?